src/main/java/uk/ac/susx/tag/classificationframework/featureextraction/pipelines/FeatureExtractionPipeline.java

```diff
@@ -42,6 +42,7 @@
 import uk.ac.susx.tag.classificationframework.featureextraction.inference.featureselection.FeatureSelector;
 import uk.ac.susx.tag.classificationframework.featureextraction.normalisation.TokenNormaliser;
 import uk.ac.susx.tag.classificationframework.featureextraction.tokenisation.Tokeniser;
+import weka.core.Stopwords;
```

Comment:
Is this import necessary?

Reply:
i dont think so i think this one was from Qiwei's work cause it was there when i started i didnt need to import anything for stop words i just added the list